build.sbt

```diff
@@ -115,11 +106,25 @@ val publishSettings = List(
   )
 )
 
-val root = (project in file("."))
+val root = tlCrossRootProject
+  .aggregate(
+    text,
+    csv,
+    csvGeneric,
+    json,
+    jsonCirce,
+    jsonPlay,
+    jsonDiffson,
+    jsonInterpolators,
+    xml,
+    scalaXml,
+    cbor,
+    cborJson,
+    finiteState
+  )
   .settings(commonSettings)
   .enablePlugins(ScalaUnidocPlugin, SiteScaladocPlugin, NanocPlugin, GhpagesPlugin)
   .settings(
-    name := "fs2-data",
     publishArtifact := false,
     publish / skip := true,
     ScalaUnidoc / unidoc / unidocProjectFilter := inAnyProject -- inProjects(
```

Comment:
Rather than excluding, my preference is to explicitly include. Then at least the list doesn't grow so big :)
https://typelevel.org/sbt-typelevel/site.html#how-can-i-publish-unidoc-api-docs